lib/wasix/src/state/env.rs

```diff
@@ -327,6 +154,12 @@ pub struct WasiEnv {
     /// Implementation of the WASI runtime.
     pub runtime: Arc<dyn Runtime + Send + Sync + 'static>,
 
+    /// The handle of the current module in the Linker. If the module is
+    /// statically linked, this will always be `MAIN_MODULE_HANDLE` which is
+    /// zero. Non-zero handles will correspond to the module's entry in the
+    /// linker, which will live in the `WasiEnv::inner` field.
+    pub dl_handle: ModuleHandle,
```

Comment:
Why a zero rather than Option<>?

Reply:
I need to remove this! Thanks for the reminder 😄